setup.py

```diff
@@ -9,4 +9,4 @@
       author_email='geodesy@ga.gov.au',
       license='Apache License 2.0',
       packages=['geodepy'],
-      install_requires=['numpy'])
+      install_requires=['numpy', 'scipy', 'gdal'])
```

Comment:
I noticed two things:

1) dependencies between the travis test environment and setup.py are not synchronised. It's possible for the tests to work, but the user downloading the code to have broken code. Solution: have some sort of install script for both (I'll have a think)

2) there aren't any tests to catch this problem. Solution: add some kind of system test which installs the setup.py and sees if that works.